src/listeners/ready.ts

```diff
@@ -16,8 +14,6 @@ export class UserEvent extends Listener {
         this.printBanner();
         this.printStoreDebugInformation();
         await sendMessage(BOT_ADMIN_LOG, { content: 'online' });
-        // await testScheduler();
-        await checkBirthdayScheduler();
```

Comment:
where is this call now? is this not needed anymore?

Reply:
The plugin imports it automatically

Reply:
![image](https://user-images.githubusercontent.com/52933626/225017979-98cd230b-f444-4ff8-ad35-f5ebf9228597.png)